todoist.el

```diff
@@ -278,6 +279,18 @@ CACHE is optional param to get projects from cache."
          (name (completing-read "Select project: " (mapcar 'todoist--project-name projects))))
     (-find (lambda (x) (equal (todoist--project-name x) name)) projects)))
 
+(defun todoist--refile-handler ()
```

Comment:
Oh, one more thing: should this also remove the `TODOIST_ID` and and `TODOIST_PROJECT_ID` properties after deleting from todoist? I didn't think about it until now, but it seems like an obvious thing to do.

Reply:
Yeah that's reasonable.